.gitignore

```diff
@@ -41,7 +41,7 @@ tests/.ipynb_checkpoints/*
 examples/.ipynb_checkpoints/*
 examples/*.pdf
 *log
-*dat
+#*dat
```

Comment:
Do not do this. You can use --force at the command line to add files.

Reply:
Sure. I will remove the comment mark then.